src/module-3/online-store/scss/base/_typography.scss

```
@@ -0,0 +1,88 @@
+:root {
+  --blue-marguerite: #6f64f8;
+  --heavy-metal: #2c2c2c;
+  --mountain-mist: #999999;
```

```suggestion
  --mountain-mist: #999;
```  
please use short hexes